src/main/resources/wa-task-configuration-wa-wacasetype.dmn

```diff
@@ -510,7 +510,7 @@ else "roleAssignmentId"</text>
           <text>"nextHearingDate"</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_1ycw69t">
-          <text>if caseData.nextHearingDate != null then caseData.nextHearingDate else ""</text>
+          <text>if (taskAttributes != null and taskAttributes.taskType = "endToEndTask" and taskAttributes.nextHearingDate != null) then "" else if (caseData.nextHearingDate != null) then caseData.nextHearingDate else ""</text>
```

Comment:
can we move this to the endToEndTask entry for  "hearingDate" instead? That way we can leave the general purpose entry  and any legacy tests using that as is.

Reply:
The dmn is set up with hearingDate being usable by any taskType so this will be the first entry by setup rule. Resolved.